roles/sft-server/templates/sftd.service.j2

```diff
@@ -23,6 +23,9 @@ ExecStart={{ sftd_bin_path }} \
             -A '{{ sft_public_ip }}' \
             -M '{{ sftd_incoming_ip }}' \
             -r '{{ sftd_metrics_port }}' \
+{% if sft_allowed_client_versions is defined %}
+            -b "{{ sft_allowed_client_versions }}" \
+{% endif %}
```

Comment:
```suggestion
            {% if sft_allowed_client_versions is defined -%}
            -b "{{ sft_allowed_client_versions }}" \
            {% endif -%}
```
This might be a bit less distracting. ([docs](https://jinja.palletsprojects.com/en/2.11.x/templates/#whitespace-control))

Reply:
Yes, that's cleaner 👌